master/docs/spelling_wordlist.txt

```diff
@@ -876,6 +876,7 @@ workdir
 Workdir
 workdirs
 workername
+workersrcs
```

Comment:
This work must not be added to list of English words exceptions. Instead, since this is not a regular word, but code, you need to wrap `workersrcs` in changelog text with double backticks:
```
``workersrcs``
```